DataStructures.Tests/UnitTest1.cs

```diff
@@ -9,7 +9,7 @@ public class LinkedListTests
         public void LinkedList_starts_empty()
         {
             // Act
-            LinkedList list = new LinkedList();
+            LinkedListTests list = new LinkedListTests();

```

Comment:
This will not work.  You need the using at the top of the file in order to create a LinkedList.  LinkedListTests doesn't allow you to do what you need to.